cds_dojson/marc21/fields/books/base.py

```diff
@@ -362,16 +377,21 @@ def accelerator_experiments(self, key, value):
 @filter_values
 def urls(self, key, value):
     """Translates urls field."""
-    try:
-        clean_val('y', value, str, manual=True)
-    except ManualMigrationRequired as e:
-        e.subfield = 't'
-        raise e
+    sub_y = clean_val('y', value, str)
     url = clean_val('u', value, str, req=True)
-    if 'cds.cern.ch' not in url:
+    eitems = self['_migration']['eitems_proxy']
+    if 'https://cds.cern.ch/auth.py' in url and sub_y == 'ebook':
```

Comment:
do we have to keep this? Chat IRL?

Reply:
I would leave it like this, since it has to be handled on the ILS side. WDYT?